### ISSUE TITLE
Images cropped to square aspect ratio

### ISSUE THREAD

**Issue by ressy** [stevenschobert/instafeed.js#461]
Apologies if this was already covered in a recent issue, but I couldn't find any quite matching this behavior.  About three weeks ago I set up Instafeed on a site and had it pulling in images at their original aspect ratio, as intended.  Then about a week later it mysteriously switched to all square images, cropping out content from the original images.

Other than `accessToken`, `clientId`, and `userId` the only options I'm using in `Instafeed()` are:

            get: user,
            sortBy: most-recent,
            limit: 5,
            resolution: low_resolution

When I looked at the image URLs Instafeed is pulling in, I noticed it's specifying some kind of bounding box options that look responsible for the cropping.  (No idea if that means Instagram is forcing the issue when it generates a URL for the client, or if Instafeed is doing that spontaneously, though.)  The relevant URL segments are things like `s320x320` and `c0.134.1080.1080`, where if I manually remove the latter in an address bar it fetches the uncropped image just as I want to see in the feed automatically.  Any ideas?

**Comment by ressy** [comment]
Looking around a little more I just saw [this pull request](https://github.com/stevenschobert/instafeed.js/pull/459), which is basically what my feed is already doing on its own for some reason.  The [referenced stackoverflow post](http://stackoverflow.com/questions/33541162/how-to-get-the-cropped-image-from-instagram-api/33545449#33545449) describes `cX.Y.W.H` being the cropping arguments, which makes sense given the  `c0.134.1080.1080` I'm seeing on my side.  Still no idea why it's happening, though.

**Comment by stevenschobert** [comment]
Hey @ressy! Have you double checked your settings in the Instagram API, to ensure that you have the option to enable non-square media checked?

**Comment by ressy** [comment]
Sorry for the delayed reply (and thanks for the quick one).  Yes, I have that check box checked under Manage Client > Migrations in the API interface.

**Comment by stevenschobert** [comment]
@ressy ah good! How did you generate the access token you're using?

**Comment by ressy** [comment]
I originally used this: http://instagram.pixelunion.net/

(I had found that referenced a few times in other issues here, so I gave it a go, and it seemed to work.)

I also tried to make sense of [Instagram's authentication instructions](https://www.instagram.com/developer/authentication/), but without success until just this moment.  I just finally realized the redirect URI referenced in the docs must be listed in the security settings for the client for it to work.  Now I just set the redirect URI arbitrarily to 127.0.0.1 and manually copied the text out of the address bar when the redirect was attempted, and I've got a shiny new access token.  But, the issue persists even with the new token in place in my `new Instafeed()` call.

Thanks again for your help.
